client/src/components/cluster.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import {
+  Flex,
+  Link,
+  Icon,
+  Text,
+  Spacer,
+  HStack,
+  Divider,
+  Button,
+} from "@chakra-ui/react";
+import { FaPlus } from "react-icons/fa";
+import CAvatar from "./clickableAvatar";
+
+const Cluster = ({ data, color }) => {
+  const follow = () => alert("Followed!");
+  const { name, users, defaultCluster } = data;
+  return (
+    <>
+      <Divider />
+      <Flex mt="2vh" mb="1vh" width="full" align="center" direction="row">
+        <Text ml="8" fontSize="xl">
+          {name}
+        </Text>
+        <Spacer />
+        {defaultCluster ? (
+          ""
+        ) : (
+          <Button
```

Comment:
We'll need `follow`, `unfollow`, `join`, `leave` buttons- based on several conditions- let's discuss them

Reply:
Yeah so the follow button will become unfollow if and when the user is already following it, we need to wait for the API for that. @utkarsh867 let us know when you're available.